### PR TITLE
Extract script

### PR DESCRIPTION
**Pull request by ayeshaa63**
Extract script created which returns a pandas dataframe. Test file created as well as conftest file.

### CODE REVIEW

File: etl-script/extract.py [hunk 1]
```diff
@@ -2,12 +2,37 @@
 The data should then be returned as a Dataframe."""
 
 import pandas as pd
+import requests
 
+NO_OF_PLANTS = 60
```
Comment (NMcKittie) [inline etl-script/extract.py:7]: Probably shouldnt hard code this

File: etl-script/extract.py [hunk 2]
```diff
@@ -2,12 +2,37 @@
...
+
+    except Exception:
+        return {'error': 'Cannot connect to the API.'}
+
+
+def get_dataframe() -> pd.DataFrame:
+    '''Puts all plant information into a dataframe.'''
+    plants = []
+
+    for i in range(1, 10):
```
Comment (NMcKittie) [inline etl-script/extract.py:29]: is this just for testing?

**Review by NMcKittie — APPROVED** [review]
Theres a few things that are hard coded that im guessing that you used for testing

File: etl-script/extract.py (hunk 1 above)
Comment (ayeshaa63) [reply etl-script/extract.py:7]: Yes I will change it

File: etl-script/extract.py (hunk 2 above)
Comment (ayeshaa63) [reply etl-script/extract.py:29]: Yes my bad changing it now

File: etl-script/extract.py [hunk 3]
```diff
@@ -2,12 +2,35 @@
...
+
+def get_dataframe(no_of_plants: int) -> pd.DataFrame:
+    '''Puts all plant information into a dataframe.'''
+    plants = []
+
+    for i in range(no_of_plants):
+        plant = get_plant_data(i)
+        plants.append(plant)
+
+    return pd.DataFrame.from_dict(plants)
```
Comment (NMcKittie) [inline etl-script/extract.py]: return a list of dictionaries instead, so this can be handled in transform

**Review by NMcKittie — APPROVED** [review]
Looks good

**Review by NMcKittie — APPROVED** [review]
Looks Good

**Pull request merged**